data/remnant/remnant missions.txt

```diff
@@ -4634,12 +4628,412 @@ mission "Remnant: Expanded Horizons Storms 1"
+			`	She shakes her head. "No, antimatter would not be a potent enough weapon. But we do have a theory involving the natural lifecycle of supergiant stars. As you may know, supernovae are triggered when the pressure from a massive star's fusion processes and the degeneracy pressure from its core can no longer match the gravitational force holding the star together. As the core collapses, runaway fusion processes are initiated and the outer envelope of the star is shed, while the core either explodes or collapses into a compact stellar object, such as a neutron star or black hole."`
+			`	"Now, main sequence stars lack a large degenerate core to detonate, so at worst they produce novae, in which the outer envelope of the star is shed, but the core remains intact as a white dwarf. Likewise, massive but young stars also lack degenerate matter in their core, so they cannot undergo a supernova until they have aged for millions of years, producing and compressing heavier elements."`
+			`	"But if a sufficient quantity of degenerate matter, somehow mined from a compact object such as a neutron star or white dwarf, were to be introduced into the star, it could upset the delicate balance between its outward fusion pressure and inward gravitational pressure, perhaps leading to a gravitational collapse event that would result in a supernova, or at least a powerful nova. Do you follow me?"`
+			choice
+				`	"Yes... I think."`
+				`	"Not really."`
+					goto simpler
+
+			label speculation
+			`	"More speculatively, if a civilization was capable of locally altering physical constants, they could simply make fusion within the star impossible, or increase the gravitational constant to initiate collapse. Or even alter the strong nuclear force to cause every atomic nucleus to decay instantly. But these would be godlike abilities, and while the Archons may be on par with that level, the Korath are certainly not. So, the lower bound for star-destroying weapons is somewhere near the technological level of the Korath at their peak."`
+			`	"The only issue with this theory is that the amount of degenerate matter one would need to introduce would be immense, on the level of multiple planetary masses. As it turns out, stars are very large," Dawn smiles for a moment, "and it takes proportionally large forces to affect them in any meaningful away within a reasonable timespan."`
+			`	"We think that perhaps some other form of exotic matter could have a similar effect, or perhaps there is an unknown way to 'process' degenerate matter to enhance its ability to initiate core collapse in a star. Or maybe our models are entirely wrong, and through stellar processes we don't yet understand, it might be possible to prematurely detonate small degenerate cores in a way that does not lead to a fully-fledged supernova, but still sufficiently tears a star apart. But enough of this. We should probably depart soon, if we are to return in a timely manner."`
```

Comment:
```suggestion
			`	"We think that perhaps some other form of exotic matter could have a similar effect, or perhaps there is an unknown way to 'process' degenerate matter to enhance its ability to initiate core collapse in a star. Or maybe our models are entirely wrong, and through stellar processes we don't yet understand, it might be possible to prematurely detonate small degenerate cores in a way that does not lead to a fully-fledged supernova, but still sufficiently tears a star apart. But enough of this. We should probably depart soon, if we are to return in a timely manner.`
```